### PR TITLE
Fix indentation and general code fixes

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: snippets/standard/io/io-exceptions/vb/io-exceptions.vb [hunk 1]
```diff
@@ -11,7 +11,7 @@ Module Program
     End Sub
 
    Function OpenStream(path As String) As StreamWriter
-        If path is Nothing Then
+        If path Is Nothing Then
```
Comment (Youssef1313) [inline snippets/standard/io/io-exceptions/vb/io-exceptions.vb]: Capitalized keyword.

File: snippets/standard/serialization/is-serializable/vb/library.vb [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
-      Public Function IsSerializable(ByVal obj As Object) As Boolean
-    		If obj Is Nothing Then Return False 
 
-			Dim t As Type = obj.GetType()
-			Return t.IsSerializable
-		End Function
-	End Module
+    Public Module UtilityLibrary
+        <Extension>
+        Public Function IsSerializable(obj As Object) As Boolean
```
Comment (Youssef1313) [inline snippets/standard/serialization/is-serializable/vb/library.vb]: Removed `ByVal`

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb [hunk 3]
```diff
@@ -2,30 +2,29 @@
...
 
-		' Create unique and foreign key constraints.
-		Dim uniqueConstraint As UniqueConstraint = New _
-            UniqueConstraint(customersTable.Columns("CustomerID"))
-		Dim fkConstraint As ForeignKeyConstraint = New _
-            ForeignKeyConstraint("CustOrdersConstraint", _
-            customersTable.Columns("CustomerID"), _
-            ordersTable.Columns("CustomerID"))
+            ' Create unique and foreign key constraints.
+            Dim uniqueConstraint As New _
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb]: Simplified variable declaration.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb [hunk 4]
```diff
@@ -2,30 +2,29 @@
...
+                UniqueConstraint(customersTable.Columns("CustomerID"))
+            Dim fkConstraint As New _
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb]: Simplified variable declaration.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb [hunk 5]
```diff
@@ -2,30 +2,29 @@
...
-	Catch ex As Exception
-		' Process exception and return.
-        Console.WriteLine("Exception of type {0} occurred.", _
-        ex.GetType().ToString())
-	End Try
-End Sub
-' </Snippet1>
+        Catch ex As Exception
+            ' Process exception and return.
+            Console.WriteLine($"Exception of type {ex.GetType().ToString()} occurred.")
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb]: Used string interpolation.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb [hunk 6]
```diff
@@ -2,57 +2,55 @@
...
+            ' Create Orders table.
+            Dim ordersTable As New DataTable("Orders")
+            ordersTable.Columns.Add("CustID", Type.GetType("System.Int32"))
+            ordersTable.Columns.Add("CustName", Type.GetType("System.String"))
+            AddHandler ordersTable.Constraints.CollectionChanged, _
+                New System.ComponentModel.CollectionChangeEventHandler( _
+                AddressOf Collection_Changed)
+
+            ' Create unique constraint.
+            Dim constraint As New _
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb]: Simplified variable declaration.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb [hunk 7]
```diff
@@ -2,57 +2,55 @@
...
+            ordersTable.Constraints.RemoveAt(0) 
  
-	Catch ex As Exception
-		' Process exception and return.
-        Console.WriteLine("Exception of type {0} occurred.", _
-            ex.GetType().ToString())
-	End Try
+        Catch ex As Exception
+            ' Process exception and return.
+            Console.WriteLine($"Exception of type {ex.GetType().ToString()} occurred.")
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb]: Used string interpolation.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb [hunk 8]
```diff
@@ -2,57 +2,55 @@
...
 
-Private Shared Sub Collection_Changed( sender As object, _
-    ex As System.ComponentModel.CollectionChangeEventArgs)
-	Console.WriteLine("List_Changed Event: '{0}'\t element={1}", _
-        ex.Action, ex.Element)
-End Sub
-' </Snippet1>
+    Private Shared Sub Collection_Changed(sender As object, _
+        ex As System.ComponentModel.CollectionChangeEventArgs)
+        Console.WriteLine($"List_Changed Event: '{ex.Action}'{vbTab} element={ex.Element}")
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb]: Used string interpolation and replaced `\t` with `vbTab`

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.Contains Example/VB/source.vb [hunk 9]
```diff
@@ -2,22 +2,21 @@
...
+        Try
+            If constraints.Contains(constraint.ConstraintName) Then
+                If constraints.CanRemove(constraint)
+                    constraints.Remove(constraint.ConstraintName)
+                End If
+            End If
+    
+        Catch e As Exception
+            ' Process exception and return.
+            Console.WriteLine($"Exception of type {e.GetType().ToString()} occurred.")
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.Contains Example/VB/source.vb]: Used string interpolation

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 10]
```diff
@@ -2,75 +2,75 @@
 
 Public Class SamPle
 
-Shared Sub Main()
+    Shared Sub Main()
 
-	' create a DataSet with two tables
-	Dim dataSet As New DataSet()
+        ' Create a DataSet with two tables.
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Follow coding conventions.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 11]
```diff
@@ -2,75 +2,75 @@
...
 
-	' create Customer table
-	Dim customersTable As New DataTable("Customers")
-	customersTable.Columns.Add("customerId", _
-        Type.GetType("System.Int32")).AutoIncrement = true
-	customersTable.Columns.Add("name",       _
-        Type.GetType("System.String"))
-	customersTable.PrimaryKey = New DataColumn() _
-        { customersTable.Columns("customerId") }
+        ' Create Customer table.
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Follow coding conventions.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 12]
```diff
@@ -2,75 +2,75 @@
...
+        Dim customersTable As New DataTable("Customers")
+        customersTable.Columns.Add("customerId", _
+            Type.GetType("System.Int32")).AutoIncrement = True
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Capitalized True

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 13]
```diff
@@ -2,75 +2,75 @@
...
-	Dim ordersTable As New DataTable("Orders")
-	ordersTable.Columns.Add("orderId",    _
-        Type.GetType("System.Int32")).AutoIncrement = true
-	ordersTable.Columns.Add("customerId", _
-        Type.GetType("System.Int32"))
-	ordersTable.Columns.Add("amount",     _
-        Type.GetType("System.Double"))
-	ordersTable.PrimaryKey = New DataColumn() _
-        { customersTable.Columns("orderId") }
+        ' Create Orders table.
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Follow coding conventions.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 14]
```diff
@@ -2,75 +2,75 @@
...
+        Dim ordersTable As New DataTable("Orders")
+        ordersTable.Columns.Add("orderId",    _
+            Type.GetType("System.Int32")).AutoIncrement = True
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Capitalized True

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 15]
```diff
@@ -2,75 +2,75 @@
...
 
-	' create Order Details table
-	Dim orderDetailsTable As New DataTable("OrderDetails")
-	orderDetailsTable.Columns.Add("orderId",    _
-        Type.GetType("System.Int32"))
-	orderDetailsTable.Columns.Add("productId",     _
-        Type.GetType("System.Int32"))
-	orderDetailsTable.Columns.Add("quantity",     _
-        Type.GetType("System.Double"))
+        ' Create Order Details table.
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Follow coding conventions.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 16]
```diff
@@ -2,75 +2,75 @@
...
-	Dim table As DataTable
-	Dim column As DataColumn
-	For Each table in dataSet.Tables 
-		Console.WriteLine(table.TableName )
-		For Each column In table.Columns 
-			Console.WriteLine(String.Format("{0} ", column.ColumnName))
-		Next
-		Console.WriteLine()
-	Next
+        ' Print the tables and their columns.
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Follow coding conventions.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 17]
```diff
@@ -2,75 +2,75 @@
...
+        Dim table As DataTable
+        Dim column As DataColumn
+        For Each table in dataSet.Tables 
+            Console.WriteLine(table.TableName)
+            For Each column In table.Columns 
+                Console.WriteLine("${column.ColumnName} ")
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Used string interpolation

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb [hunk 18]
```diff
@@ -2,75 +2,75 @@
...
-		Next
-		Console.WriteLine()
-	Next
+        ' Print the tables and their columns.
+        Dim table As DataTable
+        Dim column As DataColumn
+        For Each table in dataSet.Tables 
+            Console.WriteLine(table.TableName)
+            For Each column In table.Columns 
+                Console.WriteLine($"{column.ColumnName} ")
```
Comment (Youssef1313) [inline snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData DataRelationCollection.AddRange Example/VB/source.vb]: Used string interpolation.

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb (hunk 3 above)
Comment (sharwell) [reply snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb]: 💭 I'd almost prefer `UniqueConstraint(` move up a line

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb (hunk 7 above)
Comment (sharwell) [reply snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb]: `.ToString()` isn't necessary with string interpolation

```suggestion
            Console.WriteLine($"Exception of type {ex.GetType()} occurred.")
```

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb (hunk 8 above)
Comment (sharwell) [reply snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb]: 💭 Does `\t` work in VB?

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.Contains Example/VB/source.vb (hunk 9 above)
Comment (sharwell) [reply snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.Contains Example/VB/source.vb]:
```suggestion
            Console.WriteLine($"Exception of type {e.GetType()} occurred.")
```

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb (hunk 5 above)
Comment (sharwell) [reply snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.AddRange Example/VB/source.vb]:
```suggestion
            Console.WriteLine($"Exception of type {ex.GetType()} occurred.")
```

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb (hunk 8 above)
Comment (Youssef1313) [reply snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.CollectionChanged Example/VB/source.vb]: @sharwell, It doesn't.

**Pull request merged**